### PR TITLE
Add Linting and GitHub Actions Test

### PR DESCRIPTION
**Pull request by feng-eric**
- Fixed issue with linting
- Removed usages of redux 
- Added in unstated-next
- Added PR template

### CODE REVIEW

**Comment by feng-eric** [comment]
There is an issue with some files not being caught by the commit linting but when `yarn lint` is ran, it does catch it and fix it. Probably need to investigate this further later.

**Comment by feng-eric** [comment]
> There is an issue with some files not being caught by the commit linting but when `yarn lint` is ran, it does catch it and fix it. Probably need to investigate this further later.

Issue has been fixed. @mahadzaryab1, it would be good if you could test it yourself as well prior to us merging this in.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+### Notion ticket link
+
+<!-- Please replace with your ticket's URL -->
+
+[Ticket Name](https://www.notion.so/uwblueprintexecs/086c1b564dbf478faa649a8880e34b81?v=9d3f8002a0d1477ba395616dfc082af7)
```
Comment (mahadzaryab1) [inline .github/pull_request_template.md:5]: What is the purpose of this?

File: README.md [hunk 2]
```diff
@@ -86,11 +86,11 @@ React Hooks library for data fetching from Vercel
 
 [Go To Documentation](https://swr.vercel.app/)
 
-### **React Redux**
+### **Unstated-Next**
```
Comment (mahadzaryab1) [inline README.md:89]: Do we need to have this bit for every plugin? I think it would be better if we remove it to make our README a bit cleaner. Let me know if you want to do this in a separate PR though.

**Review by mahadzaryab1 — CHANGES_REQUESTED** [review]
Just added a few small comments/questions

File: .github/pull_request_template.md (hunk 1 above)
Comment (feng-eric) [reply .github/pull_request_template.md:5]: This is the template that other projects on Blueprint are using. The ticket name is a link back to the task on the Notion dev board.

File: README.md (hunk 2 above)
Comment (feng-eric) [reply README.md:89]: This was just to remove every instance of Redux. The task for updating the README is still not yet to be done yet.

Comment (feng-eric) [reply README.md:89]: Yes, it should be done in a separate PR. I believe I made it a task on the dev board but it's not assigned to anyone currently.

**Review by mahadzaryab1 — APPROVED** [review]
LGTM
